library-core/src/main/java/com/mikepenz/fastadapter/items/SimpleItemList.java

```diff
@@ -0,0 +1,174 @@
+package com.mikepenz.fastadapter.items;
+
+import com.mikepenz.fastadapter.IItem;
+import com.mikepenz.fastadapter.adapters.IItemList;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+
+public class SimpleItemList<Item extends IItem> extends IItemList<Item> {
+  private final List<Item> list;
+
+  public SimpleItemList() {

```

Comment:
I dont think we have to emulate the array list here, just create an List<Item> constructor and as an default one `new ArrayList<Item>();`

Reply:
👍